solutions/landing-zone-v2/README.md

```diff
@@ -0,0 +1,585 @@
+    --target-google-apis-bundle=all-apis \
+    --service-directory-registration=projects/$PROJECT_ID/locations/$REGION
+
+    # private dns zone for googleapis.com
+    gcloud dns managed-zones create googleapis \
+    --description="dns zone for googleapis" \
+    --dns-name=googleapis.com \
+    --networks=$NETWORK \
+    --visibility=private
+
+    gcloud dns record-sets create googleapis.com. --zone="googleapis" --type="A" --ttl="300" --rrdatas="10.255.255.254"
+
+    gcloud dns record-sets create *.googleapis.com. --zone="googleapis" --type="CNAME" --ttl="300" --rrdatas="googleapis.com."
+
+    # private dns zone for gcr.io
+    gcloud dns managed-zones create gcrio \
+    --description="dns zone for gcrio" \
+    --dns-name=gcr.io \
+    --networks=$NETWORK \
+    --visibility=private
+
+    gcloud dns record-sets create gcr.io. --zone="gcrio" --type="A" --ttl="300" --rrdatas="10.255.255.254"
+
+    gcloud dns record-sets create *.gcr.io. --zone="gcrio" --type="CNAME" --ttl="300" --rrdatas="gcr.io."
+    ```
+
+1. Create firewall rules
+    ```bash
+    # Allow egress to AZDO (optionnal) - should be revised periodically - https://learn.microsoft.com/en-us/azure/devops/organizations/security/allow-list-ip-url?view=azure-devops&tabs=IP-V4#ip-addresses-and-range-restrictions
+    gcloud compute firewall-rules create allow-egress-azure --action ALLOW --rules tcp:22,tcp:443 --destination-ranges 13.107.6.0/24,13.107.9.0/24,13.107.42.0/24,13.107.43.0/24 --direction EGRESS --priority 5000 --network $NETWORK --enable-logging
+
+    # Allow egress to Github (optionnal) - should be revised periodically - https://api.github.com/meta
+    gcloud compute firewall-rules create allow-egress-github --action ALLOW --rules tcp:22,tcp:443 --destination-ranges 192.30.252.0/22,185.199.108.0/22,140.82.112.0/20,143.55.64.0/20,20.201.28.151/32,20.205.243.166/32,102.133.202.242/32,20.248.137.48/32,20.207.73.82/32,20.27.177.113/32,20.200.245.247/32,20.233.54.53/32,20.201.28.152/32,20.205.243.160/32,102.133.202.246/32,20.248.137.50/32,20.207.73.83/32,20.27.177.118/32,20.200.245.248/32,20.233.54.52/32 --direction EGRESS --priority 5001 --network $NETWORK --enable-logging
+
+    # Allow egress to internal, peered vpc and secondary ranges
+    gcloud compute firewall-rules create allow-egress-internal --action ALLOW --rules=all --destination-ranges 192.168.0.0/16,172.16.0.128/28,10.0.0.0/8 --direction EGRESS --priority 1000 --network $NETWORK --enable-logging
+     
+    # Deny egress to internet
+    gcloud compute firewall-rules create deny-egress-internet --action DENY --rules=all --destination-ranges 0.0.0.0/0 --direction EGRESS --priority 65535 --network $NETWORK --enable-logging
+    ```
+
+1. Create anthos config controller cluster
+    ### GKE Autopilot - Fully managed cluster
+    TODO: TBD
+
+    ### GKE Standard
+    ```bash
+    gcloud anthos config controller create $CLUSTER --location $REGION --network $NETWORK --subnet $SUBNET 
```

Comment:
Should we add a note about limiting access to specific CIDR ranges using the `--man-block` flag? Outllined [here](https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup#standard)

Reply:
@cartyc note added, please mark as resolved if you like it